backend/cfg/cfg_equivalence.ml

```diff
@@ -483,7 +483,8 @@ let check_terminator_instruction :
     match expected.desc with
```

Comment:
Cannot attach it properly (outside of the diff), but before line 480,
there should be a `Poll_and_jump`, `Poll_and_jump` case.